<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViMainMenu</class>
 <widget class="QWidget" name="ViMainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="7">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="creationHeading">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Creation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ViButton" name="createProjectButton"/>
   </item>
   <item row="5" column="1" colspan="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="ViButton" name="projectPlaybackButton"/>
   </item>
   <item row="3" column="3">
    <widget class="ViButton" name="linePlaybackButton"/>
   </item>
   <item row="4" column="3">
    <widget class="ViButton" name="filePlaybackButton"/>
   </item>
   <item row="2" column="4" rowspan="3">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="3">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="ViButton" name="analyseWaveButton"/>
   </item>
   <item row="2" column="7">
    <widget class="ViButton" name="correlateProjectButton"/>
   </item>
   <item row="2" column="6" rowspan="3">
    <widget class="Line" name="line3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="playbackHeading">
     <property name="text">
      <string>Playback</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="analysisHeading">
     <property name="text">
      <string>Analysis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="correctionHeading">
     <property name="text">
      <string>Correction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="ViButton" name="correctProjectButton"/>
   </item>
   <item row="3" column="1">
    <widget class="ViButton" name="metadataButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViButton</class>
   <extends>QToolButton</extends>
   <header>vibutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
